pkgs/development/tools/profiling/oprofile/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, binutils, popt, zlib, pkgconfig
+{ stdenv, fetchurl, binutils, popt, zlib, pkgconfig, linuxPackages
```

Comment:
Why do you take the entire package set as an argument if `kernelHeaders` is the only item from that set that you actually need?